entry_types/scrolled/package/spec/frontend/useMediaEventsContext-spec.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {useMediaEventsContext, MediaEventsContextDataProvider} from 'frontend/useMediaEventsContext';
+
+import 'support/fakeBrowserFeatures';
+import {renderHookInEntry} from 'support';
+
+describe('useMediaContext', () => {
+
+  it('returns an object with current section and chapter index', () => {
```

Comment:
This isn't what the test tests for. Consider using `renderEntry` and maybe `simulateScrollingIntoView` to really test for *current* section and chapter index

Reply:
[changeSectionEvent-spec](https://github.com/codevise/pageflow/blob/master/entry_types/scrolled/package/spec/frontend/features/changeSectionEvent-spec.js) is also testing the same thing.